code/modules/clothing/suits/armor.dm

```diff
@@ -297,6 +297,10 @@
 /obj/item/clothing/suit/armor/riot/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/adjust_fishing_difficulty, 5)
+	init_rustle_component()
+
+/obj/item/clothing/suit/armor/riot/init_rustle_component()
```

Comment:
brother did you test this? this code doesn't compile still. there's no base proc

Reply:
wait it was tested how

Reply:
Now it compiles